controller/reviews.controller.js

```diff
@@ -32,3 +33,21 @@ exports.getAllReviews = (req, res, next) => {
     res.status(200).send({ reviews });
   });
 };
+
+exports.getCommentsById = (req, res, next) => {
+  const { review_id } = req.params;
+  selectReviewById(review_id)
+    .then((review) => {
+      if (!review) {
+        return Promise.reject;
```

Comment:
Very small suggestion. You could call promise.reject with an error message like so:

`Promise.reject(status: 404, msg:'review doesn't exist')`

Reply:
This way if you've got a custom error handler it can deal with it for you :)